### PR TITLE
Creation of Ryan Stewart profile

### PR DESCRIPTION
**Pull request by ryanstwrt**
I created a new profile for myself and updated the index.html file. Hopefully I did everything correctly. Let me know if you need anything else.

### CODE REVIEW

File: users/stewryan/index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+For summer 2017, I worked in conjunction with Idaho National Lab (INL) and ISU on developing and implementing Functional Expansion Tallies (FETs) in the neutronics code Shift for the Exascale Computing Project. The FETs are meant to create a seamless transition for data from neutronics to computational fluid dynamics codes to allow for fully coupled reactor physics problems to be solved. A short paper on th work can be found [here](./files/Report.pdf).
```
Comment (albeanth) [inline users/stewryan/index.md]: Just a couple grammatical things here. 

"For **the** summer **of** 2017..."

"..short paper on **the** work can be found..."

File: users/stewryan/index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+title: Ryan-personal
+layout: defaul
```
Comment (albeanth) [inline users/stewryan/index.md]: Missed a "t"

**default**

**Comment by albeanth** [comment]
@ryanstwrt  looks good man. Just little things... 

@rtrp/admins

File: index.html [hunk 3]
```diff
@@ -44,6 +44,7 @@
 <li><a href="{{ site.url }}users/grechanp/">Pavel Grechanuk</a> (<a href="https://github.com/grechasneak" class="user-mention">@grechasneak</a>)</li>
 <li><a href="{{ site.url }}users/schicklr/">Robert Schickler</a> (<a href="https://github.com/albeanth" class="user-mention">@robertschickler</a>)</li>
 <li><a href="{{ site.url }}users/tamashia/">Aaron Tamashiro</a> (<a href="https://github.com/aarontama" class="user-mention">@aarontama</a>)</li>
+<li><a href="{{ site.url }}users/stewryan/">Ryan Stewart</a> (<a href="https://github.com/ryanstwrt" class="user-mention">@ryanstwrt</a>)</li>
```
Comment (harterj) [inline index.html]: Change the order to last name alpha please

File: index.html [hunk 4]
```diff
@@ -44,6 +44,7 @@
 <li><a href="{{ site.url }}users/grechanp/">Pavel Grechanuk</a> (<a href="https://github.com/grechasneak" class="user-mention">@grechasneak</a>)</li>
 <li><a href="{{ site.url }}users/schicklr/">Robert Schickler</a> (<a href="https://github.com/albeanth" class="user-mention">@robertschickler</a>)</li>
```
Comment (harterj) [inline index.html:45]: @robertschickler Unrelated note, you may want to change this re-direction from Tony's page to yours...

Comment (harterj) [reply index.html:45]: `href="https://github.com/albeanth" class="user-mention">@robertschickler</a>)</li>` is the line

File: index.html (hunk 3 above)
Comment (albeanth) [reply index.html]: it took me longer than I care to admit to realize you meant to make it alphabetically haha

Comment (ryanstwrt) [reply index.html]: Alrighty, I updated on my branch. Thanks for catching some of the spelling errors, I appreciate it. Do I need to create another pull request for this most recent update?

File: index.html (hunk 4 above)
Comment (albeanth) [reply index.html:45]: @ryanstwrt  looks good man. 

@harterj  I think it's good to go. The issue with Rob's redirect link isn't necessarily related to Ryan's page. I vote we push it through and just make sure Rob knows about his incorrect link and fix it later.

File: index.html (hunk 3 above)
Comment (albeanth) [reply index.html]: @ryanstwrt you're good man! We can see all of your most recent changes.

**Comment by harterj** [comment]
:shipit:

**Pull request merged**